### PR TITLE
Markdown linting: Fixed MD046 errors

### PR DESCRIPTION
**Pull request by jeff-matthews**
## Purpose of this pull request

This pull request (PR) fixes all "fixable" MD046 errors in the devdocs project.

There are still MD046 errors that we cannot fix. The mdl tool identifies code blocks inside nested lists as errors. For example:

```
1. Lorem ipsum

   - Lorem ipsum

      ```bash
      mkdir
      ```
```

We can fix these "errors" by removing the line between the bullet and the code block, but that makes the source less readable. Since our goal with enabling linting rules is to promote the readability of source markdown, that's not a viable option.

For now, we should merge the fixes for legitimate errors and report this issue to the mdl maintainers. Until it's fixed however, we won't be enabling this rule on production.

### CODE REVIEW

**Comment by dobooth** [comment]
The irony...

File: _checks/styles/style-rules-prod [hunk 1]
```diff
@@ -35,4 +35,4 @@ rule 'MD038'
 exclude_rule 'MD039'
 rule 'MD040'
 exclude_rule 'MD041'
-exclude_rule 'MD046'
+rule 'MD046', :style => :fenced
```
Comment (dshevtsov) [inline _checks/styles/style-rules-prod]:
```suggestion
rule 'MD046', :style => :fenced # code block style
```

File: src/guides/v2.3/config-guide/memcache/memcache_centos.md [hunk 2]
```diff
@@ -20,20 +20,33 @@ To install memcached on CentOS, perform the following tasks as a user with `root
...
+   ```bash
+   yum install -y memcached
+   ```
+
+   ```bash
+   yum install -y php-pecl-memcache
+   ```
+
+   {:.bs-callout-info}
+   The syntax of the preceding commands might depend on what package repositories you use. For example, if you use webtatic and PHP 5.6, enter <code>yum install -y php56w-pecl-memcache</code>. Use `yum search memcache|grep php` to find the appropriate package name.
```
Comment (dshevtsov) [inline src/guides/v2.3/config-guide/memcache/memcache_centos.md]:
```suggestion
   The syntax of the preceding commands might depend on what package repositories you use. For example, if you use webtatic and PHP 5.6, enter `yum install -y php56w-pecl-memcache`. Use `yum search memcache|grep php` to find the appropriate package name.
```

File: _checks/styles/style-rules-prod (hunk 1 above)
Comment (jeff-matthews) [reply _checks/styles/style-rules-prod]: I shouldn't have committed that change. We don't want to turn the rule on.

**Comment by jeff-matthews** [comment]
running tests

**Comment by jeff-matthews** [comment]
running tests

**Pull request merged**